### PR TITLE
more css styling

### PR DESCRIPTION
**Pull request by monicah79**
added work section
updated license
more development on mobile work section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -178,26 +178,23 @@ <h2>About Myself</h2>
...
-        <div class="language-heading">
-            <p>Skills</p>
+            </li>
+            <div class="divider"></div>
+        <li class="language-heading">Skills
             <img src="Images/arrow-right-icon/Normal Button/Tertiary/Icons/Union.svg" alt=""/>
-        </div>
+            </li>
+        <div class="divider"></div>
     </section>
```
Comment (1GORDON) [inline index.html:198]: Great job @monicah79 🤝

- [ ] Kindly make sure the color of your design matches the color of the design provided in the Figma template.
- Your design
color
- Figma design
Figma color.

- [ ] As stated in the project requirements, kindly implement the button interactions correctly. Kindly follow the image attached to guide you on the interactions to implement.
buttons

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @monicah79 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by monicah79** [comment]
i opened a new pull request coz i merged by mistake

**Comment by monicah79** [comment]
so what am i supposed to do?just adding the buttons

**Comment by 1GORDON** [comment]
Hi @monicah79 

- [ ] You can implement the buttons as shown in the image. When you hover the button changes to a light blue color, when pressed it changes to a dark blue color, and so on.

**Pull request closed without merging**